nixos/modules/system/activation/top-level.nix

```diff
@@ -260,10 +260,10 @@ in
     };
 
     system.extraDependencies = mkOption {
-      type = types.listOf types.package;
+      type = types.listOf types.path;
```

Comment:
Wouldn't this allow values like `"/var"`? I think we should keep making sure they're in the store at least.